gapic-common/lib/gapic/grpc/errors.rb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "googleauth"
+require "gapic/common/error"
+
+module Gapic
+  # It's called GRPCName because otherwise I get naming errors
+  # where everything referencing GRPC::Anything will try and fail to 
```

Comment:
This should only happen for code that is located within the `Gapic::GRPC::` namespace. So it should be feasible just to go through this directory and changed everything to `::GRPC::`.

Reply:
not exactly, it would be any code that is referencing `GRPC::` from `Gapic::`, so everything in `gapic-common/lib` really

Reply:
Yeah that's what I meant.